Clicker/Views/Cards/MCResultCell.swift

```diff
@@ -25,7 +25,7 @@ class MCResultCell: UICollectionViewCell {
     var percentSelected: Float!
     var highlightViewWidthConstraint: Constraint!
     var didLayoutConstraints: Bool = false
-    var missedCorrectAnswer: Bool = false
+    var showCorrectAnswer: Bool = false
```

Comment:
no need to declare this a Bool. It will be inferred

Reply:
I'll take that out. Thanks!